packages/react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/react-components",
-  "version": "15.0.0",
+  "version": "17.0.0",
```

Comment:
Also bumped two major releases here.

Reply:
This  was set in anticipation. Andrew's deprecation of Select merged ahead of me @16.0.0, my deprecation of  Pagination, also a braking change , should make 17.0.0 correct